koku/api/report/aws/provider_map.py

```diff
@@ -157,7 +167,7 @@ def __init__(self, provider, report_type):
                         'cost_units_fallback': 'USD',
                         'usage_units_key': 'unit',
                         'usage_units_fallback': 'GB-Mo',
-                        'sum_columns': ['usage', 'cost', 'infrastructure_cost', 'derived_cost'],
+                        'sum_columns': ['usage', 'cost', 'infrastructure_cost', 'derived_cost', 'markup_cost'],
```

Comment:
```suggestion
                        'sum_columns': ['usage', 'cost', 'infrastructure_cost', 'derived_cost', 'markup_costs'],
```

Same thing for the other files as well...